modules/Bio/EnsEMBL/Compara/PipeConfig/Vertebrates/LoadMembers_conf.pm

```diff
@@ -64,29 +64,10 @@ sub default_options {
     return {
         %{$self->SUPER::default_options},   # inherit the generic ones
 
-    # parameters inherited from EnsemblGeneric_conf and very unlikely to be redefined:
-        # It defaults to Bio::EnsEMBL::ApiVersion::software_version()
-        # 'ensembl_release'       => 68,
-
-    # parameters that are likely to change from execution to another:
-        # It is very important to check that this value is current (commented out to make it obligatory to specify)
-        # Change this one to allow multiple runs
-        #'rel_suffix'            => 'b',
         'division'   => 'vertebrates',
-        # 'collection' => $self->o('division'),
-        'collection' => 'vertebrates',
-
-        # names of species we don't want to reuse this time
-        #'do_not_reuse_list'     => [ 'homo_sapiens', 'mus_musculus', 'rattus_norvegicus', 'mus_spretus_spreteij', 'danio_rerio', 'sus_scrofa' ],
-        'do_not_reuse_list'     => [ ],
```

Comment:
I think I would have kept this. I know you were removing anything that is set the same as the base class, but this is one that is changed quite regularly (relative to other settings anyway)

Reply:
My approach is that you can always change it from the command line when creating the pipeline, but I can put it back the commented part if you feel it is better.